rules-engine/tests/test_rules_engine/test_examples.py

```diff
@@ -144,3 +145,59 @@ def test_average_indoor_temp(data: Example) -> None:
         data.summary.setback_hours_per_day,
     )
     assert data.summary.average_indoor_temperature == approx(avg_indoor_temp, rel=0.1)
+
+def test_ua(data: Example) -> None:
+    """
+    Test how the rules engine calculates UA from energy bills.
+
+    Pulls in data and pre-calculated results from example spreadsheets
+    and compares them to the UA calculated from that data by the
+    engine.
+    """
+    # TODO: Handle oil and propane fuel types too
+    usage_data = None
+    if data.summary.fuel_type is engine.FuelType.GAS:
+        usage_data = data.natural_gas_usage
+    else:
+        raise NotImplementedError('Fuel type {}'.format(data.summary.fuel_type))
+
+    # build Home instance - input summary information and bills
+    home = engine.Home(data.summary.fuel_type,
+                       data.summary.heating_system_efficiency,
+                       thermostat_set_point=data.summary.thermostat_set_point)
```

Comment:
Looks like there's no bug in the rules engine, you just need to add this parameter:

`initial_balance_point=data.summary.estimated_balance_point`

I'm so sorry for the false alarm!

Reply:
Thinking about this more, this is probably not a suitable fix, because we need to test that the rules engine can actually converge to the correct balance point.